### PR TITLE
Replace deprecated SKE credential endpoint

### PR DESCRIPTION
**Pull request by vicentepinto98**
Follow-up to #249

### CODE REVIEW

**Review by JorTurFer — APPROVED** [review]

File: stackit/internal/services/ske/ske_acc_test.go
Comment (joaopalet) [inline stackit/internal/services/ske/ske_acc_test.go:1]: The changes to this file are unreleated, right? I'm not sure I understood why we are stopping testing the `allow_privileged_containers` logic

File: stackit/internal/services/ske/ske_acc_test.go [hunk 1]
```diff
@@ -25,16 +25,16 @@ var clusterResource = map[string]string{
...
 	"name_min":                                         fmt.Sprintf("cl-min-%s", acctest.RandStringFromCharSet(3, acctest.CharSetAlphaNum)),
-	"kubernetes_version":                               "1.24",
-	"kubernetes_version_used":                          "1.24.17",
-	"kubernetes_version_new":                           "1.25",
-	"kubernetes_version_used_new":                      "1.25.15",
+	"kubernetes_version":                               "1.25",
+	"kubernetes_version_used":                          "1.25.16",
+	"kubernetes_version_new":                           "1.26",
+	"kubernetes_version_used_new":                      "1.26.11",
 	"allowPrivilegedContainers":                        "true",
```
Comment (joaopalet) [inline stackit/internal/services/ske/ske_acc_test.go:32]: But this is now unused I think

File: stackit/internal/services/ske/ske_acc_test.go [hunk 2]
```diff
@@ -63,15 +63,8 @@ var clusterResource = map[string]string{
 	"maintenance_end_new":                              "03:03:03+00:00",
 }
 
-func getConfig(version string, apc *bool, maintenanceEnd *string) string {
+func getConfig(version string, maintenanceEnd *string) string {
 	apcConfig := ""
```
Comment (joaopalet) [inline stackit/internal/services/ske/ske_acc_test.go:67]: And this

**Review by joaopalet — APPROVED** [review]

File: stackit/internal/services/ske/ske_acc_test.go
Comment (vicentepinto98) [reply stackit/internal/services/ske/ske_acc_test.go:1]: Yes they are unrelated, but while I was updating the acceptance test I got the error that this field was deprecated, so I just removed it

**Comment by vicentepinto98** [comment]
Closing in favor of #256

**Pull request closed without merging**